### PR TITLE
Add Angular, Vue and React code snippets to dxScrollView

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: concepts/05 Widgets/ScrollView/00 Overview.md [hunk 1]
```diff
@@ -6,39 +6,264 @@ The **ScrollView** is a widget that enables a user to scroll its content.
...
+        // ...
+    })
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScrollView
+            :use-native="false"
+            :show-scrollbar="always">  <!-- or 'onScroll' | 'onHover' | 'never' -->
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/00 Overview.md]:
```suggestion
            show-scrollbar="always">  <!-- or 'onScroll' | 'onHover' | 'never' -->
```

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 2]
```diff
@@ -21,12 +25,170 @@ To scroll the **ScrollView** content by a specified distance, call the [scrollBy
...
+    <!--TypeScript-->
+    import { ..., ViewChild } from '@angular/core';
+    import { DxScrollViewModule, DxScrollViewComponent, DxButtonModule } from 'devextreme-angular';
+    // ...
+    export class AppComponent {
+        @ViewChild(DxScrollViewComponent, { static: false }) scrollView: DxScrollViewComponent;
+        // Prior to Angular 8
+        // @ViewChild(DxScrollViewComponent) scrollView: DxScrollViewComponent;
+
+        scrollUpClick = e => {
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
        scrollUp() {
```

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 3]
```diff
@@ -21,12 +25,170 @@ To scroll the **ScrollView** content by a specified distance, call the [scrollBy
...
+            this.scrollView.instance.scrollBy(-100);
+        }
+
+        scrollDownClick = e => {
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
        scrollDown() { 
```

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 4]
```diff
@@ -21,12 +25,170 @@ To scroll the **ScrollView** content by a specified distance, call the [scrollBy
...
+    <div id="scrollUpButton"></div>
+    <div id="scrollDownButton"></div>
+    <div id="scrollViewContainer">Content</div>
+
+##### Angular
+
+    <!--HTML-->
+    <dx-button 
+        text="Scroll Up"
+        (onClick)="scrollUpClick($event)">
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
        (onClick)="scrollUp($event)">
```

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 5]
```diff
@@ -21,12 +25,170 @@ To scroll the **ScrollView** content by a specified distance, call the [scrollBy
...
+    </dx-button>
+    <dx-button
+        text="Scroll Down"
+        (onClick)="scrollDownClick($event)">
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
        (onClick)="scrollDown($event)">
```

Comment (RomanTsukanov) [reply concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]: Please rename these functions in code examples for other frameworks.

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 6]
```diff
@@ -21,12 +25,170 @@ To scroll the **ScrollView** content by a specified distance, call the [scrollBy
...
+    })
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div>
+            <DxButton
+                text="Scroll Up"
+                @click="scrollUpClick($event)">
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
                @click="scrollUp">
```

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 7]
```diff
@@ -21,12 +25,170 @@ To scroll the **ScrollView** content by a specified distance, call the [scrollBy
...
+            </DxButton>
+            <DxButton
+                text="Scroll Down"
+                @click="scrollDownClick($event)">
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
                @click="scrollDown">
```

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 8]
```diff
@@ -39,49 +201,468 @@ To scroll the content both vertically and horizontally, call the [scrollBy(dista
...
+    <!--HTML-->
+    <div id="scrollButton"></div>
+    <div id="scrollViewContainer">Content</div>
+
+##### Angular
+
+    <!--HTML-->
+    <dx-button 
+        text="Scroll"
+        (onClick)="scrollClick($event)">
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
        (onClick)="scroll($event)">
```

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 9]
```diff
@@ -39,49 +201,468 @@ To scroll the content both vertically and horizontally, call the [scrollBy(dista
...
+    <!--TypeScript-->
+    import { ..., ViewChild } from '@angular/core';
+    import { DxScrollViewModule, DxScrollViewComponent, DxButtonModule } from 'devextreme-angular';
+    // ...
+    export class AppComponent {
+        @ViewChild(DxScrollViewComponent, { static: false }) scrollView: DxScrollViewComponent;
+        // Prior to Angular 8
+        // @ViewChild(DxScrollViewComponent) scrollView: DxScrollViewComponent;
+
+        scrollClick = e => {
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
        scroll() {
```

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 10]
```diff
@@ -39,49 +201,468 @@ To scroll the content both vertically and horizontally, call the [scrollBy(dista
...
+            <!-- Scrolls the content to the element with the "end" id -->
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
            // Scrolls the content to the element with the "end" id
```

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 11]
```diff
@@ -39,49 +201,468 @@ To scroll the content both vertically and horizontally, call the [scrollBy(dista
...
+                    width={100}
+                    direction="vertical">
+                    {/* Here goes long content */}
+                    <div id="end"></div>
+                </ScrollView>
+            );
+        }
+
+        scrollClick = (e) => {
+            {/* Scrolls the content to the element with the "end" id */}
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
            // Scrolls the content to the element with the "end" id
```

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 12]
```diff
@@ -39,49 +201,468 @@ To scroll the content both vertically and horizontally, call the [scrollBy(dista
...
+            DxScrollView
+        },
+        data() {
+            return {
+                myScrollViewRef
+            }
+        },
+        methods: {
+            scrollClick(e) {
+                <!-- Scrolls the content to the element with the "end" id -->
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
                // Scrolls the content to the element with the "end" id
```

File: concepts/05 Widgets/ScrollView/10 Handle Scroll Gestures.md [hunk 13]
```diff
@@ -12,36 +16,308 @@ The **ScrollView** raises the [pullDown](/api-reference/10%20UI%20Widgets/dxScro
...
+    <dx-scroll-view
+        [height]="500"
+        (onScroll)="scroll($event)">
+    </dx-scroll-view>
+
+    <!--TypeScript-->
+    import { DxScrollViewModule } from 'devextreme-angular';
+    // ...
+    export class AppComponent {
+        scroll = (e) => {
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/10 Handle Scroll Gestures.md]:
```suggestion
        scroll(e) {
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/ScrollView/00 Overview.md [hunk 14]
```diff
@@ -39,6 +193,77 @@ If simulated scrolling is used, you can specify when to show the scrollbar. For
...
+        ],
+        // ...
+    })
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScrollView
+            :use-native="false"F
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/00 Overview.md]:
```suggestion
            :use-native="false"
```

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 15]
```diff
@@ -68,20 +511,158 @@ To scroll the content to a specific element, call the [scrollToElement(targetLoc
...
+                    height={200}
+                    width={100}
+                    direction="vertical">
+                    {/* Here goes long content */}
+                    <div id="end"></div>
+                </ScrollView>
+            );
+        }
+
+        scrollClick = () => {
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
        scroll = () => {
```

File: concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md [hunk 16]
```diff
@@ -68,20 +511,158 @@ To scroll the content to a specific element, call the [scrollToElement(targetLoc
...
+
+        get scrollView() {
+            return this.scrollViewRef.current.instance;
+        }
+
+        render() {
+            return (
+                <Button
+                    text="Scroll"
+                    onClick={this.scrollClick}>
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ScrollView/05 Scroll the Content/20 API.md]:
```suggestion
                    onClick={this.scroll}>
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

**Review by RomanTsukanov — APPROVED** [review]

**Review by jsdmitry — APPROVED** [review]

**Pull request merged**